source/guide/3.2/getting-started-prerequisites.md

```diff
@@ -52,15 +53,15 @@ As a general recommendation for the average system, Cloudify would require at le
 
 ### Network
 
-The Manager must be accessible via the following ports:
+The Manager listens on the following ports:
+
+* port 80 - REST API and UI. This port should be accessible when SSL is not enabled.
+* port 443 - REST API and UI. This port should be accessible when SSL is enabled.
+* port 8101 - REST API. This port is used for internal access and as such should only be accessible by [Agent]({{page.terminology_link}}#agent) VMs.
```

Comment:
"accessible by" --> "accessible from". also more consistent with the way you wrote it later on